.env.example

```diff
@@ -0,0 +1,17 @@
+# The endpoint of the server's API that the client application will communicate with
+# E.g., http://localhost:5000/posts
+NEXT_PUBLIC_HOSTED_URL="http://localhost:3000"
+
+# The base URL for the Asgardeo root organization's API
+# E.g., https://api.asgardeo.io/t/your-org
+NEXT_PUBLIC_ASGARDEO_BASE_ORGANIZATION_URL="https://api.asgardeo.io/t/guardioinc"
+
+# The client ID for the Asgardeo Shared Application
```

Comment:
Not sure whether shared application makes sense to the end user. maybe adding a ref to https://wso2.com/asgardeo/docs/guides/organization-management/manage-b2b-organizations/share-applications/ will be helpful.

Reply:
Fixed